src/cli/cli.ts

```diff
@@ -62,25 +75,13 @@ export class Cli {
                 type: 'list',
                 choices: ['redis', 'sqlite']
             }, {
-                name: 'verifyAuthServer',
-                message: 'Will you be authenticating users from a different host?',
-                type: 'confirm'
-            }, {
                 name: 'authHost',
-                message: 'Enter the host of your authentication server.',
-                when: function(options) {
-                    return options.verifyAuthServer;
-                }
-            }, {
-                name: 'verifyAuthPath',
-                message: 'Is this the right endpoint for authentication /broadcasting/auth?',
-                type: 'confirm'
+                default: 'http://localhost',
+                message: 'Enter the host of your Laravel authentication server.',
             }, {
-                name: 'authPath',
+                name: 'authEndpoint',
```

Comment:
@barryvdh Do you think that we should omit this from init? Seems like it will most commonly be set to that path for Laravel devs. Thoughts?